src/entities/judgingVote.ts

```diff
@@ -49,92 +49,162 @@ export class JudgingVote extends BaseEntity {
+
+    const calibrationCount: { [id: string]: number } = {};
+    let calibrationComplete = false;
+    const remainingVotes = [...votes];
+
+    // Iterate over all votes
+    for (let i = 0; i < votes.length; i += 1) {
+      if (!calibrationComplete) {
+        // Not all teams have been calibrated
+        // Select the team with the lowest calibration count
+        const team = teams.sort((teamA, teamB) => ((calibrationCount[teamA.id] || 0) <= (calibrationCount[teamB.id] || 0) ? -1 : 1))[0];
+
+        // Retrieve all remaining votes where the selected team is evaluated
+        const teamVotes = remainingVotes.filter((voteToCompare) => voteToCompare.currentTeam === team.id || voteToCompare.previousTeam === team.id);
+
+        // Sort team votes by the team with the lowest calibration count
+        const vote = teamVotes.sort((a, b) => {
+          const otherTeamIdA = a.currentTeam !== team.id ? a.currentTeam : a.previousTeam;
+          const otherTeamIdB = b.currentTeam !== team.id ? b.currentTeam : b.previousTeam;
+          return (calibrationCount[otherTeamIdA] || 0) <= (calibrationCount[otherTeamIdB] || 0) ? -1 : 1;
+        })[0];
+
+        // Get team details for other team
+        const otherTeamId = vote.currentTeam !== team.id ? vote.currentTeam : vote.previousTeam;
+        const otherTeam = teams.find((teamToCompare) => teamToCompare.id === otherTeamId);
+
+        // Remove that vote from the remaining votes
+        remainingVotes.splice(votes.indexOf(vote), 1);
+
+        // Increment calibration counts for both teams
+        calibrationCount[team.id] = calibrationCount[team.id] ? calibrationCount[team.id] + 1 : 1;
+        calibrationCount[otherTeam.id] = calibrationCount[otherTeam.id] ? calibrationCount[otherTeam.id] + 1 : 1;
+
+        // CALIBRATION SCORING
+        const teamScore = scores[team.id] || { id: team.id, score: 0 };
+        const otherTeamScore = scores[otherTeam.id] || { id: otherTeam.id, score: 0 };
+
+        // Apply score impact based on vote outcome
+        const currentTeamScoreImpact = vote.currentTeamChosen ? calibrationScoreImpact : -calibrationScoreImpact;
+        teamScore.score += currentTeamScoreImpact;
+        otherTeamScore.score += -currentTeamScoreImpact;
+
+        // Update scores for both teams
+        scores[team.id] = teamScore;
+        scores[otherTeam.id] = otherTeamScore;
+
+        // Check to see if all teams have been calibrated
+        const allTeamsAreInCalibration = Object.keys(calibrationCount).length === teams.length;
+        const allTeamsAreCalibrated = !Object.values(calibrationCount).some((value) => value < votesNeededForCalibration);
+        if (allTeamsAreInCalibration && allTeamsAreCalibrated) {
+          calibrationComplete = true;
+          // Update min/max score for traditional scoring
+          const calibratedScores = Object.values(scores);
+          for (let j = 0; j < calibratedScores.length; j += 1) {
```

Comment:
@johnkahn if I did a `forEach` here, I got some super obscure error: `Function declared in a loop contains unsafe references to variable(s)` which violates the [`no-loop-func` Eslint rule](https://eslint.org/docs/rules/no-loop-func). After reading through the description, I'm still not sure what the issue was but I fixed it just in case  ¯\\_(ツ)\_/¯

Reply:
Oh and this is necessary to avoid the potential for `minScore` and `maxScore` values being totally wrong in the first few non-calibration passes. No idea why the results weren't wildly broken when this wasn't here because the first few votes judges were always the min/max scores so their score impact was virtually nothing (1st place team vs last team).